docs/architectural-decision-records/024-remove-protobufs.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Remove protocol buffers from the application
+
+---
+## ADR 0024 — Remove protocol buffers from the application
+
+- Status: Proposed
+- Date: 5/23/2023
+
+This ADR overrides our previous decision from [ADR 009](./009-data-serialization-framework.md) to handle serialized data in the application via protocol buffers. 
+
+## Decision Drivers
+- The reason protobuf was introduced to the application was for data storage reasons. We no longer have that technical requirement and plan to remove protobuf from our postgres tables (see [ADR 023](./023-seperate-contract-rates-tables-postgres)). 
```

Comment:
"for data storage reasons" isn't very clear to me, and even checking 009 doesn't really help a ton.  Can this be spelled out a bit?

Reply:
addressed in [abd2f9e](https://github.com/Enterprise-CMCS/managed-care-review/pull/1699/commits/abd2f9e49f262ce10253ba2112180c9d88e92b55)